.github/workflows/CI.yml

```diff
@@ -42,25 +36,27 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Setup Conda
+      - name: Setup Mambaforge
         uses: conda-incubator/setup-miniconda@v2
         with:
-          auto-update-conda: true
-          python-version: ${{ matrix.python-version }}
-          auto-activate-base: false
+            miniforge-variant: Mambaforge
+            miniforge-version: latest
+            activate-environment: test
+            use-mamba: true
 
       - name: Set cache date
         run: echo "DATE=$(date +'%Y%m%d')" >> $GITHUB_ENV
 
       - uses: actions/cache@v2
+        env:
+          CACHE_NUMBER: 0
         with:
           path: ${{ matrix.prefix }}
           key: ${{ matrix.label }}-conda-${{ hashFiles('ci/environment.yml') }}-${{ env.DATE }}-${{ env.CACHE_NUMBER }}
         id: cache
 
       - name: Update environment
-        run: conda env update -n test -f ci/environment.yml
-        if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
I believe the issue was with this `if` statement

Reply:
Yeah, I think that too. I removed it in the other PR